python/docs/source/getting_started/install.rst

```diff
@@ -129,17 +129,17 @@ PySpark is included in the distributions available at the `Apache Spark website
 You can download a distribution you want from the site. After that, uncompress the tar file into the directory where you want
 to install Spark, for example, as below:
 
-.. code-block:: bash
+.. parsed-literal::
 
-    tar xzvf spark-3.4.0-bin-hadoop3.tgz
+    tar xzvf spark-\ |release|\-bin-hadoop3.tgz
```

Comment:
Please note that. Like the snapshot shows, it will be `master` by default and Apache Spark release script injects the version number via RELEASE_VERSION environment variable.

![Screenshot 2023-06-27 at 9 21 23 AM](https://github.com/apache/spark/assets/9700541/a5a5ed98-c37e-47c4-ba14-69923c50dfd7)